### ISSUE TITLE
Multiple YubiKeys with same HMAC-SHA1 challenge response mode

### ISSUE THREAD

**Issue by createyourpersonalaccount** [Yubico/yubikey-manager#482]
- YubiKey Manager (ykman) version: 4.0.8-dev0
- How was it installed?: dnf package manager
- Operating system and version: Fedora linux 5.15.16-200.fc35.x86_64
- YubiKey model and version: YubiKey 5 NFC firmware version 5.4.3
- Bug description summary: No option to program two yubikeys with same key for HMAC-SHA1 challenge response mode.
 
Steps to reproduce
---

I have two YubiKeys. I use `ykman --device XXX otp chalresp -g 2` then attempt to copy the key with `ykman --device YYY otp chalresp 2`. The second command requires a key; if I copy/paste the previous secret key it won't work. I've also tried to convert the hex key into binary data and feed `ykman` the binary data via a pipe but I get a utf8 encoding error. 

Expected result
---

I was hoping that the pipe `ykman --device YYY otp chalresp 2 < mykey` would work; or that there would be a way to feed a hex key.

Actual results and logs
---

```
$ cat mykey | ykman otp chalresp -t 2
Error: 'utf-8' codec can't decode byte 0xc7 in position 0: invalid continuation byte
```

Other info
---

If this is a missing feature for `ykman`, is there interest in developing it? I could give it a go.

**Comment by dainnilsson** [comment]
> The second command requires a key; if I copy/paste the previous secret key it won't work.

This should work. What happens when you try it?

**Comment by createyourpersonalaccount** [comment]
@dainnilsson You are right, I have just tested that this works. The way I tested it was by feeding the same message into the challenge-response calculation:

```
$ ykman --device XXX otp calculate 2 00 # send the 00 message
<output>
$ ykman --device YYY otp calculate 2 00
<same output>
```

My bad! Thank you for the help. I came here from an [XY problem](https://en.wikipedia.org/wiki/XY_problem) with KeePassXC, assuming that my issue was my inability to program the same key into the hmac-sha1 mode...
